### DISCUSSION TITLE
是不是可以增加一个查看打开MC还要多久的功能🤔

### DISCUSSION THREAD

**Discussion by XiaoLianZhenJun** [Meloong-Git/PCL#2747] [提问] [answered]
可以在启动时显示，这样可以让使用者知道大概还要等多久，更好规划时间

**Comment by Pigeon0v0** [comment]
这要怎么预计...

**Comment by burningtnt** [comment]
这个是做不到的，如果硬要加，加个假进度条得了

**Comment by so1ve** [comment]
这个好做，卡99%就行了😁

**Comment by WForst-Breeze** [comment, marked as answer]
没法预测吧，有个进度条已经差不多足够了，毕竟大部分情况下真正费时间的部分不在于启动器启动游戏，在于那个游戏启动后 Minecraft 自身的资源加载进度（也就是那个黑底或者红底白色进度条），要规划时间主要看的也是这个进度，而不是启动进度

**Comment by Chiloven945** [comment]
如果这个能做出来的话现在所有的软件问题都会迎刃而解，因为这代表着你完全可以预测任何的软件操作

**Reply by MoYuan-CN** [reply]
但是并不可能.png
